bin/node/src/commands/genesis/mod.rs

```diff
@@ -195,6 +199,14 @@ fn parse_auth_inputs(
     }
 }
 
+fn parse_storage_mode(storage_type: &str) -> Result<AccountStorageType> {
+    match storage_type.to_lowercase().as_str() {
+        "on-chain" => Ok(AccountStorageType::OnChain),
+        "off-chain" => Ok(AccountStorageType::OffChain),
+        lowercase_type => Err(anyhow!("The provided value for storage_type ({}) does not match the expected values (on-chain, off-chain)", lowercase_type))
```

Comment:
minor nit: could we inline the variable into the string? (e.g., "The provided value for storage_type ({lowercase_type}) does not match the expected values (on-chain, off-chain)".

Also, should we rename this to just `mode`?

Reply:
done!